core/src/main/java/io/questdb/griffin/engine/functions/lt/LtDoubleVVFunctionFactory.java

```diff
@@ -62,9 +63,16 @@ public FuncVV(Function left, Function right) {
 
         @Override
         public boolean getBool(Record rec) {
```

Comment:
We need to port this change to JIT compiler - otherwise, Java and JIT filters may produce different results.